xml/rmt_install.xml

```diff
@@ -141,6 +141,716 @@
+       *"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>: schedule for running the
+       <command>rmt-cli sync</command> command (syncing with the &suse;
+       database)
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.image.pullPolicy</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>: <literal>"Always"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>:
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.image.repository</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>:
+       <literal>"registry.suse.de/suse/sle-15-sp3/update/bci/containerfile/suse/rmt-server"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>: &rmt; server image
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.image.tag</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>: <literal>"2.7"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>: &rmt; server image tag
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.init.image.pullPolicy</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>: <literal>"Always"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>:
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.init.image.repository</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>:
+       <literal>"registry.suse.de/suse/sle-15-sp3/update/bci/containerfile/suse/rmt-mariadb-client"</literal>
+      </para>
+      <para>
+       <emphasis role="strong">Description</emphasis>:
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><emphasis role="strong">Key</emphasis>: app.init.image.tag</term>
+     <listitem>
+      <para>
+       <emphasis role="strong">Type</emphasis>: string
+      </para>
+      <para>
+       <emphasis role="strong">Default</emphasis>: <literal>"10.5"</literal>
```

Comment:
I would suggest to use `latest` as the tag, as once SLE 15 SP4 comes out, we'll switch the mariadb container to 10.6 and 10.5 will cease to receive updates.

WDYT @brunoleon?